nilearn/glm/tests/test_first_level.py

```diff
@@ -573,6 +575,197 @@ def test_first_level_glm_computation_with_memory_caching():
         del mask, func_img, FUNCFILE, model
 
 
+@pytest.mark.parametrize('t_r, warning_msg',
+                         [(None, "No bold.json .* BIDS")])
+def test_first_level_from_bids_set_repetition_time_warnings(tmp_path,
+                                                            t_r,
+                                                            warning_msg):
+
```

Comment:
The naming is great, but i'd not be against a short docstring...

Reply:
added some explanations: let me know if that's enough